server/odc-server/src/main/resources/log4j2.xml

```diff
@@ -15,11 +15,15 @@
              如果每个类的 log 对象都基于当前类生成，%logger 可作为 %class 的替代。
         -->
         <Property name="LOG_PATTERN">
-            [%date{yyyy-MM-dd HH:mm:ss.SSS}][%thread][%X{traceId},%X{spanId},%X{requestId}][%level][%class][%line]: %message%n
+            [%date{yyyy-MM-dd HH:mm:ss.SSS}][%thread][%X{traceId},%X{spanId},%X{requestId}][%level][%class][%line]:
```

Comment:
here seems only a line breaker change, let's check the xml code format configuration